Sources/MapboxMaps/Location/Puck/Puck2D.swift

```diff
@@ -16,7 +20,11 @@ internal final class Puck2D: Puck {
                         return true
                     }
                     .add(to: cancelables)
+                if let pulsing = configuration.pulsing, pulsing.isEnabled {
```

Comment:
wdyt about this syntax instead?
```suggestion
                if configuration.pulsing?.isEnabled == true {
```